cookbook/ui_customization/add_a_tab_to_product_edit_form.rst

```diff
@@ -0,0 +1,209 @@
+How to add a tab or a panel to the product edit form
+====================================================
+
+The most common UI customization on the Akeneo PIM is to add a tab to the product edit form. With the new product edit form introduced in 1.4 we splitted tabs in distinct elements:
+
+ - Tabs hold all edit features on the entity (attributes edit, classification, associations)
+ - Panels hold all meta information about the entity (history, completeness, comments)
+
+Add a tab to the product edit form
+----------------------------------
+
+Let's say that we would like to add a custom tab to our product edit form in order to manage packages of the product.
+
+First, we need to create a Form extension in our bundle:
+
+.. code-block:: javascript
+    :linenos:
+
+    'use strict';
+    /*
+     * /src/Acme/Bundle/EnrichBundle/Resources/public/js/product/form/packages.js
+     */
+    define(['pim/form'],
+        function (BaseForm) {
+            return BaseForm.extend({
+                configure: function () {
+                    this.trigger('tab:register', {
+                        code: this.code,
+                        isVisible: this.isVisible.bind(this),
+                        label: 'Packages tab'
```

Comment:
not for the first example. I want to keep this example as simple as possible and using a translation key add a dependency (underscore)